### ISSUE TITLE
Screen Share broken on Win10

### ISSUE THREAD

**Issue by heye** [StrikeOrg/ulterius-server#77]
The Screen Share seems to be broken, here is what I did (on two different machines, both running windows 10):

-  Downloaded current version (1.9) from ulterius.io
-  Installed
-  Open Client via localhost and also on a different machine with local ip address
-  Open Screen Share an click connect 
-  The server instantly  starts using a lot of cpu (25%-38%) and keeps reserving memory until within secons the maxium of about 8GB is reached, then it frees most of it and starts again

I also tried to reboot, restart ulteries, disable gpu, and change the framerate.

here is part of the server log:

_Logs Ready
1.9.2.0
Loading Webcams
Setting up HTTP Server
Web Server is running on this port: 22006
Creating system service
Service: False56 packet handlers loaded!
Cron Daemon Started
Api Server started at 0.0.0.0
Starting Terminal API
Checking for Updates
Terminal Server bound to 0.0.0.0:22008
Trying to forward ports
Connection from 127.0.0.1:51120
UlteriusServer.TerminalServer.Messaging.TerminalControl.Requests.AesHandshakeRequest
Manager started for 22009
Manager started for 22010
No NAT Device Found
Started Monitor on 1
[Topshelf] Started
Hardware Survery Completed
Disconnection from 127.0.0.1:51121
Connection from 127.0.0.1:51168
Manager started for 22010
Manager started for 22009
UlteriusServer.TerminalServer.Messaging.TerminalControl.Requests.AesHandshakeRequest
Screen Share Died
Disconnection from 127.0.0.1:51171
Screen Share Died
Connection from 127.0.0.1:51713
Manager started for 22010
Manager started for 22009
Connection from 192.168.5.9:54710
Manager started for 22010
Manager started for 22009
UlteriusServer.TerminalServer.Messaging.TerminalControl.Requests.AesHandshakeRequest
Disconnection from 127.0.0.1:51799
Screen Share Died
Connection from 127.0.0.1:51999
Manager started for 22010
Manager started for 22009
Disconnection from 127.0.0.1:51999
Connection from 127.0.0.1:52003
Disconnection from 127.0.0.1:52000
Connection from 127.0.0.1:52004
Connection from 127.0.0.1:52005
Packet failed to decrypt: Der Wert darf nicht NULL sein.
Parametername: s
Disconnection from 127.0.0.1:52003
Connection from 192.168.5.155:52037
Manager started for 22010
Manager started for 22009
UlteriusServer.TerminalServer.Messaging.TerminalControl.Requests.AesHandshakeRequest_

There are also a lot of these unobserved task exceptions:
0:{} 13 keys
NativeErrorCode:6
ClassName:"System.ComponentModel.Win32Exception"
Message:"Das Handle ist ungültig"
Data:null
InnerException:null
HelpURL:null
StackTraceString:" bei System.Drawing.Graphics.CopyFromScreen(Int32 sourceX, Int32 sourceY, Int32 destinationX, Int32 destinationY, Size blockRegionSize, CopyPixelOperation copyPixelOperation)
 bei System.Drawing.Graphics.CopyFromScreen(Int32 sourceX, Int32 sourceY, Int32 destinationX, Int32 destinationY, Size blockRegionSize)
 bei AgentInterface.Api.ScreenShare.ScreenData.CaptureDesktop() in D:\Documents\StrikeTaskManager\AgentInterface\Api\ScreenShare\ScreenData.cs:Zeile 358.
 bei UlteriusServer.Api.Network.PacketHandlers.ScreenSharePacketHandler.HandleFullFrame() in D:\Documents\StrikeTaskManager\RemoteTaskServer\Api\Network\PacketHandlers\ScreenSharePacketHandler.cs:Zeile 385.
 bei System.Threading.Tasks.Task.Execute()"
RemoteStackTraceString:null
RemoteStackIndex:0
ExceptionMethod:"8 CopyFromScreen System.Drawing, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a System.Drawing.Graphics Void CopyFromScreen(Int32, Int32, Int32, Int32, System.Drawing.Size, System.Drawing.CopyPixelOperation)"
HResult:-2147467259
Source:"System.Drawing"
WatsonBuckets:null

**Comment by andrewmd5** [comment]
I'm not too sure, is the service enabled properly? Really we just need to do a patch to address a ton of these issues.

**Comment by heye** [comment]
I tried both options, GPU enabled/disabled with the same result.

**Comment by heye** [comment]
How do I correctly enable the service? I can start/stop it in the taskmanager and it is running after a reboot. The Daemon Manager shows server and agent as running.

**Comment by andrewmd5** [comment]
And with GPU enabled does the same error happen? Also is this a VM or a physical pc?

**Comment by heye** [comment]
Yes it show the same high-memory-loop behaviour and it is a physical pc. I also tried to build the application with vs for debugging but cannot find the WindowsInput project. Here is a screenshot from the server log:

<img width="800" alt="screen shot 2017-09-15 at 09 44 39" src="https://user-images.githubusercontent.com/3592376/30471870-985ea78e-99fa-11e7-80f5-d7384e95196e.png">

**Comment by andrewmd5** [comment]
Can you check if this issue is resolved in the latest version?
